ads/webediads.js

```diff
@@ -16,11 +16,13 @@
 
 import {loadScript, validateData} from '../3p/3p';
 
-/** Created by Webedia on 07/03/16 */
+/** Created by Webedia on 07/03/16
```

Comment:
Please update the JSdoc for the function. 
Here's an example https://github.com/ampproject/amphtml/blob/master/ads/google/doubleclick.js#L33
Also follow the format, no comment following `/**` when there're multiple lines.
```
/** 
 * Coments .....
 * @param ...
 */
```